scripts/gulp/plugins/util/parseBuffer.ts

```diff
@@ -1,6 +1,6 @@
 import { parse } from '@babel/parser';
 
-const parseBuffer = buffer =>
+const parseBuffer = (buffer: any) =>
```

Comment:
big hassle to connect all the dots, if there is a type I could use please add suggestion. ty

Reply:
purely looking at the name have you tried `Buffer` ?

Reply:
I tried following the chain up and using buffer and it seems to work, until `through2` which just uses `any` as this type 🤷

Reply:
https://github.com/microsoft/fluentui/pull/24526/commits/4da335f5778310b6dea126080d309f4627ba8ab4